kodilla-stream/src/main/java/com/kodilla/stream/immutable/ForumUser.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.stream.immutable;
+
+public final class ForumUser {
+    private final String userName;
+    private final String realName;
+
+    public ForumUser(final String userName, final String realName) {
+        this.userName = userName;
+        this.realName = realName;
+    }
+
+    public String getUserName() {
+        return userName;
+    }
+
+    public String getRealName() {
+        return realName;
+    }
+
+}
```

Comment:
To jest czesc innego zadania - czy na pewno wyszedles z mastera z tym branchem ?